discussions/no-customizing-classnames.md

```diff
@@ -1,58 +0,0 @@
-# No Customizing `className`'s
```

Comment:
Well, it's still true but we need to expose classNames so Fela can inject styles into the components. Eventually, I'm going to write some new pieces explaining the whole Fela thing.